### DISCUSSION TITLE
Want to make new voicebank

### DISCUSSION THREAD

**Discussion by HinakoIzumi** [titinko/utsu#81] [Q&A] [answered]
Thank you for amazing software!

I'd like make new voicebank  from .wav files (just recorded).
In my environment, I cannot select [File]->[New]->[Voicebank].
Is it the tool to do it? Then, how can I use this function?
If it is WIP, I'm looking forward to your releases.
Sorry for my poor English.

Thanks.

**Comment by titinko** [comment, marked as answer]
The voicebank creation tool is still WIP.

However, all you need to make a basic voicebank is WAV files plus an oto.ini text file.  If you create an empty oto.ini for your WAV files, you can then configure it in [File]->[Open Voicebank].  To find guides for how to make an oto.ini file, try googling "utau oto tutorial" and you will find some good ones.

**Reply by HinakoIzumi** [reply]
Your advice has allowed me to start working on the voicebank. I am looking forward to completing the creation tools.
Thanks.
